client/src/components/end-screen/end-screen.vue

```diff
@@ -30,6 +30,19 @@ const earnings = computed(() => {
   if (!initialBalance || !balance) return new BigNumber(0);
   return balance.minus(initialBalance);
 });
+
+const insuffientBalance = computed(() => {
+  const balance = channel?.balances.user as BigNumber;
+  return balance.isLessThan(1e18);
+});
+
+function closeChannel() {
```

Comment:
Proposing verbs to be for functions

```suggestion
function shouldCloseChannel() {
```